src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7445,6 +7445,11 @@ components:
           type: string
         updatedDate:
           type: string
+        jobId:
```

Comment:
Really interesting question.

> In the future, I could see a user hitting the get snapshot request (which doesn't currently exist) or the enumerate endpoint (which currently doesn't return this response object) and wanting to see the createdSnapshotId. 

I think the enumerate request endpoint does return a list of this response type (SnapshotAccessRequestResponse).  So, that to me is worth adding these fields to the response. Otherwise, I agree, it would just be confusing to API users.

Reply:
Ah you're right it does return a list of SnapshotAccessRequestResponse! So, in terms of my other question, do we think users will want to interact with job/flight id? Or is that something that we just want to store more for our purposes of tracking if the create flight was successful, but don't necessarily need to return to the user?

Reply:
We don't expect for the user to interact with the job id, but I think we'll want it returned to the UI on enumerating requests. For example, we could grab the status of the flight id and display it in the UI.